python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -891,6 +891,28 @@ def Index__new__(cls, *args, **kwargs):
 
 _eval_func = _FunctionProxy(_Unusable(), pd.eval)
 
+_read_pickle_func = _FunctionProxy(_Unusable(), pd.read_pickle)
+
+_to_pickle_func = _FunctionProxy(_Unusable(), pd.to_pickle)
+
+@register_proxy_func(pd.read_pickle)
+def _read_pickle(
+    filepath_or_buffer,
+    compression = "infer",
+    storage_options = None,
+):
+    return _read_pickle_func(filepath_or_buffer, compression, storage_options)
+
+
+@register_proxy_func(pd.to_pickle)
+def _to_pickle(
+    *args,
+    **kwargs,
+):
+    return _to_pickle_func(
+        *args,
+        **kwargs,
+    )
```

Comment:
Can you move this above `_eval_func` since `_eval_func` is used immediately below?